rosetta/configuration/configuration.go

```diff
@@ -103,17 +101,24 @@ func (c *Configuration) Errors() []meta.ErrorDefinition {
 
 func (c *Configuration) Check(network identifier.Network) error {
 	if network.Blockchain != c.network.Blockchain {
-		return failure.InvalidNetwork{
-			Blockchain: network.Blockchain,
-			Network:    network.Network,
-			Message:    fmt.Sprintf("invalid network identifier blockchain (have: %s, want: %s)", network.Blockchain, c.network.Blockchain),
+		return errors.InvalidNetwork{
+			Description: "invalid network identifier blockchain",
+			Details: []errors.Detail{
+				errors.WithString("blockchain", network.Blockchain),
+				errors.WithString("network", network.Network),
+				errors.WithString("blockchain_want", c.network.Blockchain),
```

Comment:
If you use `have_length` and `want_length`, then we should also use `have_blockchain` and `want_blockchain` here.